test_projects/ios/EarlGreyExample/flank.ios.yml

```diff
@@ -3,7 +3,7 @@ gcloud:
   test: ./earlgrey_example.zip
   xctestrun-file: ./dd_tmp/Build/Products/EarlGreyExampleSwiftTests_iphoneos13.4-arm64e.xctestrun
   device:
-  - model: iphone8
-    version: 12.0
+  - model: iphone13pro
+    version: 15.7
```

Comment:
Does this work for this `EarlGreyExample`?
If not, is this example needed for anything else or should it be deleted entirely?